src/main/java/tc/oc/pgm/commands/MapCommands.java

```diff
@@ -97,144 +116,113 @@ private static boolean matchesAuthor(PGMMap map, String query) {
       aliases = {"mapinfo", "map"},
       desc = "Shows information a certain map",
       usage = "[map name] - defaults to the current map")
-  public void map(Audience audience, CommandSender sender, @Text PGMMap map) {
-    MapInfo mapInfo = map.getInfo();
-    audience.sendMessage(mapInfo.getFormattedMapTitle());
-
-    MapPersistentContext persistentContext = map.getPersistentContext();
-
-    Component edition = new PersonalizedText(mapInfo.getLocalizedEdition(), ChatColor.GOLD);
-    if (!edition.toPlainText().isEmpty()) {
-      audience.sendMessage(
-          new PersonalizedText(mapInfoLabel("command.map.mapInfo.edition"), edition));
-    }
+  public void map(Audience audience, CommandSender sender, @Text MapInfo map) {
+    audience.sendMessage(
+        ComponentUtils.horizontalLineHeading(
+            ChatColor.DARK_AQUA
+                + map.getName()
+                + " "
+                + ChatColor.GRAY
+                + map.getVersion().toString(),
```

Comment:
I think this should be translatable? I guess in Arabic you should have `v1.0 Airship Battle` since they read right-to-left.

Reply:
Sure, but not a high priority.

Reply:
Leave a todo there so we don't forget it when we migrate to text library.